packages/docs/fluent-editor/demos/file-upload-handle.spec.ts

```
@@ -0,0 +1,31 @@
+import { expect, test } from '@playwright/test'
+
+const pngBuffer = Buffer.from(
```

ESLint

[error] Unexpected use of the global variable 'Buffer'. Use 'require("buffer").Buffer' instead.

(node/prefer-global/buffer)